libs/ngrx-toolkit/src/lib/shared/empty.ts

```diff
@@ -1,2 +1,2 @@
 // eslint-disable-next-line @typescript-eslint/ban-types
-export type Emtpy = {};
```

Comment:
I am not sure about that to be honest. I know you just fixed a typo there, but do we really want to call it empty. I can remember of some eslint rule which says `Empty` should be `Record<string, never>`.

Reply:
Yeah, eslint itself recommends `Record<string, never>`:
> If you want a type meaning "empty object", you probably want `Record<string, never>` instead.'

What do you think about renaming the type to `EmptyObject` to create a better alignment between its name and its usage and use `Record<string, never>` to apply eslints recommendation?

Reply:
Absolutely, would you actually create an alias for that or go directly with the `{}`?

I heard once that TypeScript is faster, when using aliases instead of "individual" types.

Reply:
Personally I prefer a type alias as it ensures a consistent code style.
I've just checked what would happen when we'd use `Record<string, never>` instead of `{}` and it would prevent state patching as expected as the Record type wouldn't allow any updates.
Maybe I'm trying to overengineer it right now... when looking at its usage it represents that either do not have any specific expectations to the store or do not add anything to state/signals/methods.

I did some research about the performance and it depends - aliases can bloat up the bundle when massively used and inlined.